revolv/administrator/forms.py

```diff
@@ -0,0 +1,27 @@
+from django import forms
+from revolv.payments.models import AdminAdjustment
+from revolv.project.models import Project
+
+class AdjustmentForm(forms.ModelForm):
```

Comment:
Just a thought, would it be easier to use [modelform_factory](https://docs.djangoproject.com/en/1.7/topics/forms/modelforms/#modelform-factory-function)?